packages/webcomponentsjs/README.md

```diff
@@ -157,6 +161,15 @@ Here's a more complicated example:
 <script type="module"></script>
 ```
 
+#### Trusted Types
+
+If you're using the loader on a page that enforces the `trusted-types` Content
+Security Policy, you'll need to allow the `webcomponents-loader` policy name so
+that the loader can dynamically create and insert a `<script>` for the polyfill
+bundle it selects based on feature detection. You should also treat
+`WebComponents.root` as if it were required to be set to a `TrustedScriptURL`
+when trusted types are being enforced.
```

Comment:
```suggestion
bundle it selects based on feature detection. If you set `WebComponents.root`
(which is rare), it should be set to a TrustedScriptURL for Trusted Types
compatibility.
```

Reply:
I'll work on the language more here. I don't think we can actually enforce that this is a `TrustedScriptURL` without making a breaking change, but I think I could word this so that the idea that it should only be set to values that would be considered as safe as a `TrustedScriptURL` was more clear. I.e. we can't actually enforce the safety ourselves, so users should know that it needs to be treated this way.

Reply:
I've updated this slightly, PTAL

Reply:
WDYT of my recommended wording here?

Reply:
Yeah, this is better - updated.